onyo/commands/history.py

```diff
@@ -78,25 +78,21 @@ def history(args: argparse.Namespace, opdir: str) -> None:
     """
     repo = None
     try:
-        repo = Repo(opdir, find_root=True)
+        repo = Repo(Path.cwd(), find_root=True)
         repo.fsck(['asset-yaml'])
     except OnyoInvalidRepoError:
         sys.exit(1)
 
     # get the command and path
-    path = sanitize_path(args.path, opdir)
+    path = sanitize_path(args.path, repo)
     history_cmd = get_history_cmd(args.interactive, repo)
 
     # run it
-    orig_cwd = os.getcwd()
     status = 0
     try:
-        os.chdir(opdir)
```

Comment:
Removing this will cause `'history` with `onyo -C` to fail.

`git log`, `tig`, etc require that they are run from within a git repository (or use `-C` ).

Reply:
It is not removed, it is moved into `main.py`, were we now generally change the cwd to opdir, so that the commands to not have to care about opdir, thereby this does not fail, because these commands are indeed run from insode a git repository.

Reply:
Cool. Thanks for explaining. :-)